README.md

```diff
@@ -1,19 +1,17 @@
-
 # AWS Build Cluster Script
 
-This script allows you to to configure and build a EC2 instances cluster machine on Amazon Web Service.
+This script allows you to to configure and build an EC2 instances cluster on Amazon Web Service.
 It arises for educational purposes to speed up and automate the creation of a cluster.
 
-An Amazon AWS account is needed, also AWS Educate should works.
-
+An Amazon AWS account is needed (AWS Educate accounts work as well).
 
 ## Dependencies
 
-To use this script you must have AWS CLI and jq 
+To use this script you must have AWS CLI and `jq` installed 
 
-1. Install the **AWS CLI**, see [here](https://docs.aws.amazon.com/cli/latest/userguide/installing.html)
-   If you don't have it, you must install Python.
-   Test if it works with the following command:
+1. Install the **AWS CLI**, see [official documentation](https://docs.aws.amazon.com/cli/latest/userguide/installing.html)
+   For AWS CLI v1, if you don't have it, you must install Python; for v2 all required tools are self-contained.
+   Test if it works by launching the following command:
```

Comment:
Here, an user may be misled if it's using AWS CLI v2 that integrates Python and is therefore not needed in the environment. Quoting from [official source](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html):
> For AWS CLI version 2, it doesn't matter if you have Python installed and if you do, it doesn't matter which version. AWS CLI version 2 uses only the embedded version of Python (and any other dependencies) that is included in the installer.